image_match/signature_database_base.py

```diff
@@ -117,7 +117,7 @@ def insert_single_record(self, rec):
         raise NotImplementedError
 
     def __init__(self, k=16, N=63, n_grid=9,
-                 crop_percentile=(5, 95), distance_cutoff=0.45,
+                 crop_percentile=(5, 95), distance_cutoff=0.45, score_cutoff=9.0,
```

Comment:
Stuff for Elasticsearch should be in the Elasticsearch driver.  You can include a **kwargs here and pick it up in the child class (tell me if it's not clear what I'm saying).